client/config/config.go

```diff
@@ -247,6 +247,16 @@ type ClientTemplateConfig struct {
 	DisableSandbox    bool
 }
 
+func (c *ClientTemplateConfig) Copy() *ClientTemplateConfig {
+	if c == nil {
+		return nil
+	}
+
+	nc := new(ClientTemplateConfig)
+	*nc = *c
```

Comment:
Do we need to deep copy the `FunctionBlacklist` as we've done with the top-level config:

```go
nc.Servers = helper.CopySliceString(nc.Servers)
```

Reply:
Indeed - good catch.